app/src/androidTest/java/com/sdp/movemeet/Activity/ActivityDescriptionTest.java

```diff
@@ -29,35 +29,35 @@
 import static androidx.test.espresso.action.ViewActions.click;
 import static androidx.test.espresso.action.ViewActions.closeSoftKeyboard;
 import static androidx.test.espresso.action.ViewActions.scrollTo;
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
 import static androidx.test.espresso.matcher.ViewMatchers.withId;
-import static com.sdp.movemeet.Sport.Soccer;
+import static com.sdp.movemeet.Sport.Soccer;;
 
 @RunWith(AndroidJUnit4.class)
 public class ActivityDescriptionTest {
 
     @Rule
     public ActivityScenarioRule<ActivityDescriptionActivity> testRule = new ActivityScenarioRule<>(ActivityDescriptionActivity.class);
 
-
-
     @Test
     public void create() {
         Intents.init();
-
+        try{
```

Comment:
Why the thread must sleep before release ?

Reply:
That was not necessary, I had just forgotten to delete it. Thx !

Reply:
You're welcome